tox.ini

```diff
@@ -54,11 +54,8 @@ commands =
     isort --check-only --diff {[vars]all_path}
     black --check --diff {[vars]all_path}
 
-[testenv:static-{charm,lib,unit,integration}]
```

Comment:
I think it's nice to have the option to run them when we want to.
Perhaps remove them from the [workflow](https://github.com/canonical/grafana-agent-k8s-operator/blob/main/.github/workflows/pull-request.yaml) instead of from tox?

Reply:
Once there is one error, though, it will just fail forever right?

Reply:
Yes, it will fail until someone decides to do something about it :)
I plan to keep using it, unless there is a compelling reason to remove it from tox in addition to the workflow.

Reply:
While there may not be a workaround. I think we decided to leave it in tox.

Reply:
Currently, `git stash pop` for 
```difff
-    mypy
+    mypy==0.950
+    httpcore==0.14.7
+    responses==0.20.0
```
is quick and dirty and works.

If we deem the tox env as necessary/useful I am happy to pin mypy for now, just like we do for many other packages.
```
find .. -type f -name "requirements*.txt" -or -name "tox.ini" -print0 | xargs -0 rg "==|>|<"
```
I think mypy 0.95 is better than no mypy. At least for now.